content/en/docs/concepts/cluster-administration/swap-memory-management.md

```diff
@@ -0,0 +1,480 @@
+---
+title: Swap memory management
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Swap is a fundamental and an invaluable Linux feature.
+It offers numerous benefits, such as effectively increasing a {{< glossary_tooltip text="node's" term_id="node" >}}
+memory by swapping out unused data,
+shielding nodes from system-level memory spikes,
+preventing Pods from crashing when they hit their memory limits,
+and [much more](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md#user-stories).
+As a result, the node special interest group within the Kubernetes project
+has invested significant effort into supporting swap on Linux nodes.
+
+<!-- body -->
+
+## In which cases swap can be valuable?
```

Comment:
@lmktfy I was hesitating to whether this fits to documentation.
What do you think? If it is I can gladly fill this section up.